.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.words": ["TANF", "checkmark", "ocap"]
```

Comment:
This file should not be in version control. I think the following commands will fix it:

```sh
git rm --cached .vscode/settings.json
git status # it should say that the settings.json file has been deleted
git add .
git commit
```